array_methods_lab.js

```diff
@@ -0,0 +1,308 @@
+// # Exercises
```

Comment:
Overall good job. I see that you are developing your array method skills . Remember to use correct syntax for function creation.